.eslintrc

```diff
@@ -4,7 +4,6 @@
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "plugin:prettier/recommended",
```

Comment:
any idea, why this this was removed?

Reply:
it caused errors on my machine, will add it later

Reply:
lets avoid removing things just because things dont work on our machines